tests/cpu/GpuShader_tests.cpp

```diff
@@ -195,6 +195,51 @@ OCIO_ADD_TEST(GpuShader, generic_shader)
 
         OCIO_CHECK_EQUAL(fragText, shaderDesc->getShaderText());
     }
+
+    {
+        OCIO_CHECK_NE(shaderDesc->getLanguage(), OCIO::GPU_LANGUAGE_GLSL_VK_4_6);
+        OCIO_CHECK_NO_THROW(shaderDesc->setLanguage(OCIO::GPU_LANGUAGE_GLSL_VK_4_6));
+        OCIO_CHECK_EQUAL(shaderDesc->getLanguage(), OCIO::GPU_LANGUAGE_GLSL_VK_4_6);
+
+        OCIO_CHECK_THROW_WHAT(shaderDesc->setDescriptorSetIndex(123, 0),OCIO::Exception, "Texture binding start index must be greater than 0.");
+        OCIO_CHECK_NO_THROW(shaderDesc->setDescriptorSetIndex(123, 456));
+        OCIO_CHECK_EQUAL(shaderDesc->getDescriptorSetIndex(), 123);
+        OCIO_CHECK_EQUAL(shaderDesc->getTextureBindingStart(), 456);
+
+        auto getSize = []() -> float { return 2; }; //simulate only 2 elements in the array
```

Comment:
This should fix the broken Windows build.

```suggestion
        auto getSize = []() -> int { return 2; }; //simulate only 2 elements in the array
```